### PR TITLE
feat(cli): upgrade to Lighthouse 6.4.1

### LINKED ISSUE #443: Lighthouse 6.3
Hi! Google's "PageSpeed Insights" is updated to version 6.3 on the 27th of august. It will be nice to get an update of "lhci" too.

### PR DESCRIPTION
**Pull request by csabapalfi**
BREAKING CHANGE: new audits

fixes #443

### CODE REVIEW

**Comment by csabapalfi** [comment]
Simplest step (bumping package version) is done. Will look at adding the new fixtures, etc shortly...

**Comment by csabapalfi** [comment]
@patrickhulce This PR is now ready for review. Please let me know what you think! 😊

File: packages/cli/test/cli.test.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ describe('Lighthouse CI CLI', () => {
   let projectAdminToken;
   let urlToCollect;
 
+  beforeAll(async () => {
```
Comment (patrickhulce) [inline packages/cli/test/cli.test.js:40]: 👍

**Review by patrickhulce — APPROVED** [review]
thank you so much @csabapalfi this is fantastic! an enormous help!!  🙏 😃 🎉

I'm going to go ahead and merge this right after I ship one more non-breaking version with the current fixes on master 👍

**Pull request merged**